binding/vue/README.md

```diff
@@ -1,48 +1,38 @@
-# rhino-web-vue
+# Rhino Binding for Vue
 
-Vue mixin for Rhino Web.
+## Rhino Speech-to-Intent engine
 
-## Rhino
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
 
-Rhino is Picovoice's Speech-to-Intent engine. It directly infers intent from spoken commands within a given context of interest, in real-time.
+Rhino is Picovoice's Speech-to-Intent engine. It directly infers intent from spoken commands within a given context of
+interest, in real-time. For example, given a spoken command:
 
-E.g. using the [demo "Clock" Rhino context (English language)](https://github.com/Picovoice/rhino/blob/master/resources/contexts/wasm/clock_wasm.rhn), Rhino performs inference on a spoken phrase:
-
-> "Set a timer for ten minutes"
+> Can I have a small double-shot espresso?
+Rhino infers that the user would like to order a drink and emits the following inference result:
 
 ```json
 {
-  "isFinalized": true,
-  "isUnderstood": true,
-  "intent": "setTimer",
+  "isUnderstood": "true",
+  "intent": "orderBeverage",
   "slots": {
-    "minutes": "10"
+    "beverage": "espresso",
+    "size": "small",
+    "numberOfShots": "2"
   }
 }
 ```
 
-> "Tell me a joke"
+Rhino is:
 
-```json
-{
-  "isFinalized": true,
-  "isUnderstood": false
-}
-```
-
-The key `isFinalized` tells you whether Rhino has reached a conclusion or is still awaiting more frames of audio to reach a decision. Upon `isFinalized=true`, `isUnderstood` will be set to true/false. If true, the `intent` will be available, as will `slots` if any were captured in this expression.
+* using deep neural networks trained in real-world environments.
+* compact and computationally-efficient, making it perfect for IoT.
+* self-service. Developers and designers can train custom models using [Picovoice Console](https://console.picovoice.ai/).
 
 ## Compatibility
 
-This library is compatible with Vue:
-- Vue.js 2.6.11+
-- Vue.js 3.0.0+
-
-The Picovoice SDKs for Web are powered by WebAssembly (WASM), the Web Audio API, and Web Workers.
-
-All modern browsers (Chrome/Edge/Opera, Firefox, Safari) are supported, including on mobile. Internet Explorer is _not_ supported.
-
-Using the Web Audio API requires a secure context (HTTPS connection) - except `localhost` - for local development.
+- Chrome / Edge
+- Firefox
+- Safari
```

Comment:
can you call this section `Browser Compatibility` and add a section called `Framework Compatibility` where you add:
```
- Vue.js 2.6.11+
- Vue.js 3.0.0+
```